### PR TITLE
W-15830591-MUnit-in-Studio-docs-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/test-with-munit-in-studio.adoc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* xref:munit::event-processors.adoc[]
+* xref:munit::munit-cookbook.adoc[]
+
+== Get to Know MUnit Tests
+
+You can perform the following tasks with MUnit in Studio:
+
+//TODO: Make the list of tasks you can perform in Studio with MUnit and create a link from each bullet to the content on this very same page
+* Create a test
+* Create a test suit
```
Comment (egrave) [inline modules/ROOT/pages/test-with-munit-in-studio.adoc]: suite

**Review by egrave — DISMISSED** [review]

File: modules/ROOT/pages/test-with-munit-in-studio.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/test-with-munit-in-studio.adoc]: Great catch! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+= Using MUnit Coverage in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can run a coverage report from Anypoint Studio.
+
+The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. +
+To run coverage reports from Maven see xref:coverage-maven-concept.adoc[].
```
Comment (dukesphere) [inline modules/ROOT/pages/coverage-studio-concept.adoc]:
```suggestion
The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. 

To run coverage reports from Maven, see xref:coverage-maven-concept.adoc[].
```

**Review by egrave — DISMISSED** [review]

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc]: OK! Thanks!

File: modules/ROOT/pages/test-with-munit-in-studio.adoc [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+The `Enable Flow Sources` element within the MUnit test tells MUnit which flow sources to load and execute for the test.
+By default, MUnit doesn't start the event sources (flow sources such as triggers and listeners). 
+
+You can define flow sources with the *Mule Properties* view of your MUnit test in Studio:
+
+. Click *Add*.
+. Select the name of the flow from *Name of the flow*.
+
+Learn more about flow sources in MUnit docs (xref:munit::enable-flow-sources-concept.adoc[]) and configure which flow sources must start for your tests.
```
Comment (dukesphere) [inline modules/ROOT/pages/test-with-munit-in-studio.adoc:90]: We have so many names for these (event sources, triggers, flow sources, and more). Let's deal with that later.

**Review by dukesphere — APPROVED** [review]

File: modules/ROOT/pages/test-with-munit-in-studio.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/test-with-munit-in-studio.adoc:90]: Sure! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
+= Using MUnit Coverage in Studio
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:1]: Is there a way to make this more user-task-focused? Like "Report on xxx Coverage Using MUnit"?

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can run a coverage report from Anypoint Studio.
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:6]: What value does the coverage report have for the user? What does it include?

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. 
+
+To run coverage reports from Maven, see xref:coverage-maven-concept.adoc[].
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:10]: Include in the previous paragraph.

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+== Enable Coverage Reports in Anypoint Studio
+
+See which flows, and which percentage of event processors in those flows the test covers by clicking on the Coverage button:
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:14]: To see..., click  *Coverage*.

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+image::using-munit-coverage-in-anypoint-studio.png["MUnit Coverage tab highlighted in the MUnit section"]
+
+The Overall Coverage value represents the percentage of the Mule application event processors that the MUnit test executes.
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:18]: Is Overall Coverage a UI element?

Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:18]: The Overall Coverage value represents the percentage of Mule application event processors that the MUnit test executes.

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report highlighted"]
+
+[%header,cols="20a,80a"]
+|===
+| Column      | Description
+| *Resource*    | Name of the Mule configuration file
+| *Containers* | Number of flows in the Mule configuration file
+| *Weight*      | Percentage of your application in a configuration file +
+                A percentage of the overall number of event processors in the documentation and the
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:30]: What does "event processors in the documentation" mean?

File: modules/ROOT/pages/coverage-studio-concept.adoc [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+Click *Generate Report* to get the report that details the flows and event processors in the Mule configuration files included in the MUnit test.
+
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report highlighted"]
+
+[%header,cols="20a,80a"]
+|===
+| Column      | Description
+| *Resource*    | Name of the Mule configuration file
+| *Containers* | Number of flows in the Mule configuration file
+| *Weight*      | Percentage of your application in a configuration file +
```
Comment (hannanelson) [inline modules/ROOT/pages/coverage-studio-concept.adoc:29]: Is the second sentence fragment elaborating on the first or is it different? It's not clear how these two sentence fragments relate to each other. Consider either combining into one or using bullets if they are two different things.

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:1]: The report is a feature that enhances API testing. Let’s see if my rewording works.
Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:6]: Included! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 6 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:10]: Done! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 7 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:14]: OK! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 8 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:18]: It's a label in the UI, so I bolded it.
Thanks!

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:18]: OK! Thanks!

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 10 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:29]: Yes, they’re related. "Weight" isn’t an easy concept. Let's see if my new sentences make it easier to understand the concept.

File: modules/ROOT/pages/coverage-studio-concept.adoc (hunk 9 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/coverage-studio-concept.adoc:30]: Deleted in the newer version. 
Thanks!

**Pull request merged**